packages/playground/src/components/Playground.tsx

```diff
@@ -11,129 +11,18 @@ import {
   UiSchema,
   ValidatorType,
 } from '@rjsf/utils';
-import localValidator from '@rjsf/validator-ajv8';
 
 import { samples } from '../samples';
+import Header from './Header';
```

Comment:
Importing `LiveSettings` when exported from `Header` prevents a circular import
```suggestion
import Header, { LiveSettings } from './Header';
```